backend/tests/features/carrinho_compras.feature

```diff
@@ -0,0 +1,31 @@
+Feature: carrinho de compras
+    As eu sou um cliente
+    I want escolher os itens que desejo comprar
+    So that eu possa efetuar e receber meu pedido
+
+Scenario: inserir um item no carrinho de compras
+    Given O array "carrinho_compras" do "users/1"
+    When "GET /menu/0" é solicitado
+    And o "GET" responde um json com os valores "ID": "0"
+    And "Name": "Batata"
+    And "Price": "10.00"
+    And "Description": "Não é a frita"
+    And "Image": "None"
+    And "Category": "Promoção"
+    And "Availability": "0 minutos"
+    And o array "carrinho de compras" adiciona o "id": "0" no ultimo índice
+    Then o array carrinho de compras tem o item "0" adicionado
+
+Scenario: remover um item no carrinho de compras
+    Given O array "carrinho_compras" do "users/1"
+    And um elemento do array tem o valor "2" 
+    When "DELETE /carrinho/2" é solicitado
```

Comment:
It's better separate the METHOD and ROUTE, then replace for
When o usuário faz uma requisição "DETELE" para a rota "/cart/2"